haystack/pipeline.py

```diff
@@ -291,6 +291,40 @@ def _overwrite_with_env_variables(cls, definition: dict):
                 param_name = key.replace(env_prefix, "").lower()
                 definition["params"][param_name] = value
 
+    def export_yaml_config(self, path: Path):
+        """
+        Save a YAML configuration for the Pipeline.
```

Comment:
"Save the configuration of the current pipeline to a YAML file so that it can later be loaded again via `load_from_yaml()` "